generic/tests/cfg/kdump.cfg

```diff
@@ -18,14 +18,14 @@
         kdump_config = core_collector makedumpfile -c -d 31
     RHEL.6:
         kernel_param_cmd = "grubby --update-kernel=`grubby --default-kernel` --args=crashkernel=128M@16M"
-        kdump_enable_cmd = "chkconfig kdump on && service kdump start"
+        kdump_enable_cmd = "chkconfig kdump on && service kdump restart"
         kdump_propagate_cmd = "service kdump propagate"
         kdump_restart_cmd = "service kdump restart"
     RHEL.7:
         kernel_param_cmd = "grubby --update-kernel=`grubby --default-kernel` --args=crashkernel=auto"
-        kdump_enable_cmd = "chkconfig kdump on && kdumpctl start"
+        kdump_enable_cmd = "chkconfig kdump on && systemctl restart kdump.service"
```

Comment:
Hi @ypu 

In my test, 120s is not enough for the cmd,, this case will be always failed due to the timeout error.

    File "/home/staf-kvm-devel/autotest-devel/client/tests/virt/test-providers.d/downloads/io-github-autotest-qemu/generic/tests/kdump.py", line 120, in kdump_enable
      session.cmd(kdump_enable_cmd, timeout=120)
    File "/home/staf-kvm-devel/autotest-devel/client/tests/virt/virttest/aexpect.py", line 1621, in cmd
      print_func)
    File "/home/staf-kvm-devel/autotest-devel/client/tests/virt/virttest/aexpect.py", line 1551, in cmd_status_output
      o = self.cmd_output(cmd, timeout, internal_timeout, print_func)
    File "/home/staf-kvm-devel/autotest-devel/client/tests/virt/virttest/aexpect.py", line 1473, in cmd_output
      raise ShellTimeoutError(cmd, o)
  ShellTimeoutError: Timeout expired while waiting for shell command to complete: 'chkconfig kdump on && systemctl restart kdump.service'    (output: "chkconfig kdump on && systemctl restart kdump.serv<chkconfig kdump on && systemctl restart kdump.servi                         \x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08\x08ce\nNote: Forwarding request to 'systemctl enable kdump.service'.\n")

Thanks.